index.html

```diff
@@ -937,20 +937,29 @@ <h2>
         </p>
       </section>
     </section>
-    <section class="informative">
+    <section>
       <h2>
         Privacy and Security Considerations
       </h2>
-      <section>
-        <h3>
-          Access to aspects of a user's local computing environment
-        </h3>
-        <p>
-          The screen orientation type and angle of the device can be accessed
-          with the API specified in this document, and can be a potential
-          fingerprinting vector.
-        </p>
-      </section>
+      <p>
+        A screen's [=current orientation type|type=] and [=current orientation
+        angle|angle=] is a potential fingerprinting vector. To resist
+        fingerprinting (e.g., in private browsing), user agents MAY:
+      </p>
+      <ol>
+        <li>Restrict the value return by the {{ScreenOrientation/type}}
+        attribute to {{OrientationType/"portrait-primary"}} or
+        {{OrientationType/"landscape-secondary"}} to match the screen's
+        aspect ratio.
+        </li>
+        <li>Always return `0` for the value of the {{ScreenOrientation/angle}}
+        attribute.
+        </li>
+        <li>If the screen orientation changes, not fire the <a data-link-for=
+        "ScreenOrientation">change</a> event to reveal a change to a
+        [=secondary=] orientation.
```

Comment:
Wouldn't this encourage battery-expensive polling instead? If you support rotating in a way that changes width/height of the viewport I think we should require that something changes. But limiting it two values for type and angle each seems reasonable.

Reply:
> Wouldn't this encourage battery-expensive polling instead?

Sorry, I might need to word this better. The answer is "no", because the UA wouldn't fire events when switching from "X-primary" to "X-secondary". Only if "X" changes, would the event fire, but it would always report as "X-primary". 

The reason the orientation change event does fire, is that the screen width/height would change, which is already observable either polling `screen`'s attribute or simply by `matchMedia("(orientation: landscape)")`.